src/Notification/ToastNotification.svelte

```diff
@@ -102,10 +105,7 @@
       <NotificationButton
         {iconDescription}
         {notificationType}
-        on:click={() => {
-          open = false;
-          dispatch('close');
-        }} />
+        on:click={close()} />
```

Comment:
The `close` function is invoked here, causing the notification to unexpectedly close.

```diff
- on:click={close()} />
+ on:click={close} />
```